### PR TITLE
Added 2 options to sync wc whitelist

### PR DESCRIPTION
**Pull request by octaedro**
Fixes the [issue 554](https://github.com/Automattic/wc-calypso-bridge/issues/554)

#### Changes proposed in this Pull Request:

This PR adds the options `woocommerce_task_list_hidden` and `woocommerce_onboarding_profile` to sync wc whitelist.

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Verify that the values of `woocommerce_task_list_hidden` and `woocommerce_onboarding_profile` are synchronized.

#### Proposed changelog entry for your changes:

* Added the options `woocommerce_task_list_hidden` and `woocommerce_onboarding_profile` to sync wc whitelist.

### CODE REVIEW

**Review by jeherve — APPROVED** [review]

**Review by timmyc — APPROVED** [review]
Thanks Fernando, this looks perfect.

**Comment by octaedro** [comment]
Hi @jeherve, thank you for your review 🙂
I wanted to ask you, how is the workflow usually in this repo? Should I `Squash and merge` this PR after the approval or is there somebody in the team that will do it later?

**Comment by jeherve** [comment]
If it's all green, you can hit the merge button! 👍

**Pull request merged**
